my_python.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+import os
+import subprocess
+import sys
+
+
+def some_func():
```

Comment:
This code is so easy to read! I've never read such simple code before. It doesn't even need comments!